frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -154,10 +155,8 @@ export const EmbedFrame = ({
   const hasVisibleParameters = visibleParameters.length > 0;
 
   const hasHeader = Boolean(finalName || dashboardTabs);
-  const isParameterPanelSticky =
-    !!dashboard &&
-    theme !== "transparent" && // https://github.com/metabase/metabase/pull/38766#discussion_r1491549200
```

Comment:
This case is fixed by making the filters background [semi transparent](https://metaboat.slack.com/archives/C063Q3F1HPF/p1718362906400589?thread_ts=1718215529.624989&cid=C063Q3F1HPF).

This is huge because now all options have the same feature parity. Previously this is a quirk that we never document. And it could caught some customers off guard. It even caught me off guard.